docs/user-guide/broker_hierarchies.md

```diff
@@ -27,7 +27,7 @@ For examples, the config JSON for the Distribution System A (where Broker A is a
 ```JSON
 {
     "name" : "DistributionSystemA",
-    "coreinitstring": "--broker_address=tcp://127.0.0.1"
```

Comment:
It looks like these sample JSONs in the  documentation for the have been reverted to use the (as of v3) old names.

Reply:
there are a lot of files that got moved around, which is why it wasn't building so I am trying to reconstruct things.  Looks like I need to do a bit more work.

Reply:
I know @allisonmcampbell has been restructuring the files associated with the User Guide; would you have any problem with her doing a review as well to make sure her changes aren't getting trampled on?